database/seed.js

```diff
@@ -0,0 +1,49 @@
+const Item = require('./database.js');
+const faker = require('faker');
+
+Item.deleteMany()
+
+for (let i = 0; i < 1; i++) {
+  let fives = Math.random() / 5;
+  console.log(fives)
+  let fours = Math.random() / 5;
+  let threes = Math.random() / 5;
+  let twos = Math.random() / 5;
+  let ones = 1 - fives - fours - threes - twos;
```

Comment:
Can you help me understand what you are trying to accomplish here?

Reply:
I mean, I know you are generating random totals for each number of star reviews, but are you wanting the numbers to be decimals < 1?

Reply:
Each star is a percentage and all the percentages add up to 100% if you run this all of these add up to 100%. This is just how I decided to do it for now. I am going to change it to a better way later.

Reply:
I am going to change a lot actually. I will eventually make it so I rather than making a property that is number of stars and then randomly generating percentages. I am going to randomly generate number of stars.

Reply:
This is just a working solution for now.

Reply:
Got it. Thanks for explaining your thought process!